documentation/docs/scenarios/changeManagement.md

```diff
@@ -0,0 +1,80 @@
+# Develop Applications Hybridly with Jenkins and SAP Solution Manager
+
+Set up an agile development process with Jenkins CI, which automatically feeds changes into SAP Solution Manager.
+
+## Prerequisites
+
+* You have downloaded the Java Runtime Environment 8.
+* You have donwloaded Jenkins 2.60.3 or higher.
+* You have set up Project “Piper”. See [README](https://github.com/SAP/jenkins-library/blob/master/README.md).
+* You have installed SAP Solution Manager 7.2 SP6. See [README](https://github.com/SAP/devops-cm-client/blob/master/README.md).
+* You have downloaded the Multi-Target Application (MTA) Archive Builder 1.0.6 or any compatible version. See [SAP Development Tools](https://tools.hana.ondemand.com/#cloud).
+
+## Context
+
+In many SAP development scenarios, it is vital to synchronize both backend and frontend deliveries. These deliveries are typically an SAP UI5 application and an ABAP backend from which it is served. The SAP UI5 parts are often developed agilely and use continuous integration pipelines that automatically build, test, and deploy the application.
+
+In this scenario, we want to show how an agile development process with Jenkins CI can automatically feed changes into SAP Solution Manager. In SAP Solution Manager, the parts of the application stack come together and can be subject to classic change and transport management.
+
+The basic workflow is as follows:
+
+1. The pipeline checks the Git commit message for a change document in status `in development`. The template for the commit message looks as follows:
```

Comment:
> checks the Git commit message

Basically our supposed workflow is like this: The feature is developed in a PR. There are n commits beween the common anchestor of the base branch and the PR head - I suppose n > 1 in typical cases. In one of the commits `origin/master..PR` (this is: in all commits reachable from `PR` but not reachable from `origin/master` there must be the change document id. It can also be provided several times - as long as the same id is provided consistently.

My point is: "the git commit message" sounds like there is exactly one commit. But infact we will have more than one commit typically. IMO "checks the Git commit messages" (plural) would be better.

Maybe some words on which commit messages are checked would also be helpful. But here I'm not sure about didactical reduction vs explain the entire thing.